chipsec_tools/edk2/PythonEFI/edk2module.c

```diff
@@ -47,7 +47,7 @@ PyDoc_STRVAR(edk2__doc__,
   // -- Access to CPU MSRs
   extern void _rdmsr( unsigned int msr_num, unsigned int* msr_lo, unsigned int* msr_hi );
   extern void _wrmsr( unsigned int msr_num, unsigned int  msr_hi, unsigned int  msr_lo );
-  extern void _swsmi( unsigned int smi_code_data, unsigned int rax_value, unsigned int rbx_value, unsigned int rcx_value, unsigned int rdx_value, unsigned int rsi_value, unsigned int rdi_value );
+  extern void _swsmi( unsigned long long smi_code_data, unsigned long long rax_value, unsigned long long rbx_value, unsigned long long rcx_value, unsigned long long rdx_value, unsigned long long rsi_value, unsigned long long rdi_value );

```

Comment:
i understand that 32bit assembly currently does not support swsmi, however the code should be future proof.  Please use the type UINTN opposed to long long.